utils/route_diag/src/main.cpp

```diff
@@ -0,0 +1,269 @@
+// Tool to test routing from one node to another node.
+//
+// This tool is intended to allow extremely verbose router output on exactly
+// one path to diagnose router behavior, e.g. look ahead and A* characteristics.
```

Comment:
Probably worth adding a "How to use" example for this utility tool

Reply:
Done